app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -145,20 +146,14 @@ class TabTrayView(
 
         view.tab_layout.addOnTabSelectedListener(this)
 
-        val tabs = if (isPrivate) {
-            view.context.components.core.store.state.privateTabs
-        } else {
-            view.context.components.core.store.state.normalTabs
-        }
+        val tabs = getTabs(isPrivate)
 
         val selectedBrowserTabIndex = tabs
             .indexOfFirst { it.id == view.context.components.core.store.state.selectedTabId }
 
-        if (tabs.size > EXPAND_AT_SIZE || startingInLandscape) {
-            expand()
-        }
+        accomodateTabs()
```

Comment:
Let's rename this to `updateBottomSheetBehavior()`

Reply:
I feel like with this naming we're offering implementation details that the caller should not be concerned about while still not clearly expressing the intent. The kdoc suggested below will help with that.

Reply:
I am thinking about this from the perspective of just reading the code without jumping to the function. This would be absolutely clear to me without jumping and looking at what `accomodateTabs()` if we added (for example):

```
// Adjust the bottom sheet height to accommodate the number of tabs to display.
updateBottomSheetBehavior()
```

Alternative, naming would be `updateTabTrayHeight`, `adjustTabTrayHeight` or some variant of that. I could see these namings without the comment needed.